functions.php

```diff
@@ -73,65 +114,73 @@ public function register_taxonomies() {
 	 * @param  wp_query.
 	 */
 	public function tags_support_query( $wp_query ) {
-		if ( $wp_query->get('tag') ) {
+		if ( $wp_query->get( 'tag' ) ) {
 			$wp_query->set( 'post_type', 'any' );
 		}
 
-		if ( $wp_query->get('category_name') ) {
+		if ( $wp_query->get( 'category_name' ) ) {
 			$wp_query->set( 'post_type', 'any' );
 		}
 	}
 
-	/**
-	 * Hook in and add a Theme metabox. Can only happen on the 'cmb2_admin_init' or 'cmb2_init' hook.
-	 */
+	// Hook in and add a Theme metabox. Can only happen on the 'cmb2_admin_init' or 'cmb2_init' hook.
```

Comment:
Function docs should use a multiline comment style. So how it was before was correct.

Reply:
Done